test/src/integration/integration_test.go

```diff
@@ -165,14 +170,36 @@ func runAnsiblePlaybooks(t *testing.T) {
 	}
 
 	// Install tools
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventory.ini", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
 
 	// Deploy scalardl
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-deploy-scalardl.yml", "-e", "base_local_directory=../../../conf"})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventory.ini", []string{"./playbooks/playbook-deploy-scalardl.yml", "-e", "base_local_directory=../../../conf"})
+}
+
+func runAnsiblePlaybooksWithGoss(t *testing.T, targetModules []string, targetHosts string) {
```

Comment:
The method name doesn't sound good to me. How about `runGoss` simply?

Reply:
Looks simple and good.👍
Fixed in a7adf28